structurizr/workspace.dsl

```diff
@@ -0,0 +1,260 @@
+						tags "Browser,Microsoft Azure - App Services"
+					}	
+				}
+			}
+
+			systemIntegration = softwareSystem "Integration Layer" "Data exchange between FO and back office" {
+				containerBoServiceBus = container "Service Bus" "Messaging platform for data exchange" "Azure ServiceBus Namespace" {
+					tags "Microsoft Azure - Azure Service Bus"
+
+					group "Commands" {
+						componentCaseCmdTopic = component "Service Bus Appeal submission topic" "Appeal submission commands"
+						componentLpaqCmdTopic = component "Service Bus LPA submission topic" "LPA submission commands"
+					}
+
+					group "Events" {
+						componentCaseMessageTopic = component "Service Bus Appeal topic" "Appeal messages"
+						componentDocumentMessageTopic = component "Service Bus Document topic" "Appeal document messages"
+						componentServiceUserMessageTopic = component "Service Bus Service User topic" "Service user messages"
+						componentEventMessageTopic = component "Service Bus Event topic" "Event message"
+					}
+				}
+			}
+
+            systemGovUk = softwareSystem "GOV Notify" {
+				tags = "ExternalSystem"
+
+				containerGovNotify = container "GOV Notify" "UK government messaging platform for sending emails, text and letters to users"
+			}
+
+			systemClamAv = softwareSystem "ClamAV" "External ClamAV defintions" {
+				tags = "ExternalSystem"				
+			}
+
+			systemDevops = softwareSystem "Azure Devops" {
+				tags = "ExternalSystem"
+
+				containerPipelines = container "Pipelines"
+				containerInfraPipelines = container "Infra Pipelines"
+			}
+
+			systemGithub = softwareSystem "Github" {
+				tags = "ExternalSystem"
+			}
+
+            systemLegacyIntegration = softwareSystem "Legacy Horizon Wrapper" "Data exchange between FO and legacy back office" "Kong" {
+				tags "LegacySystem"
+			}
+
+            systemLegacyBo = softwareSystem "Horizon" "Legacy internal service to manage planning appeals in England" {
+				tags "LegacySystem"
+			}
+
+			# Relationships
+
+			// Deployment
+			containerPipelines -> systemGithub "Retrieves code"
+			containerPipelines -> containerKeyVault "Retrieves secrets"
+			containerPipelines -> containerRegistry "Pushes new container versions to registry"
+			systemAppsFo -> containerRegistry "Apps pull latest container version"
+			containerInfraPipelines -> systemAppsFo "Terraform updates infra"
+
+			// users
+			userCaseOfficer -> containerBoWeb "Manages cases"
+			userInspector -> containerBoWeb "Decides cases"
+
+			userAppellant ->  containerFrontDoor "Registers appeals"
+            userLocalPlanningAuthority ->  containerFrontDoor "Responds to appeals"
+            userInterested -> containerFrontDoor "Comments on appeals"
+            userRule6 -> containerFrontDoor "Comments on appeals"
+
+			// Networking
+			containerFrontDoor -> containerAppealsWAF "Check traffic with WAF"
+			containerFrontDoor -> containerFoWeb "Forward valid traffic to user facing website"
```

Comment:
I'd suggest including more technical detail here to make it clearer and on Front Door itself e.g. Is it performing the SSL offloading?

e.g. `containerFrontDoor -> containerFoWeb "Forward valid traffic to user facing website" "HTTP"` (If the traffic is unencrypted at this stage)

Reply:
as far as I know it forwards the traffic and keeps the protocol used on initial request, so should be https -> https
think it must use the internal azure cert that comes with the app service